src/pseudopeople/interface.py

```diff
@@ -608,12 +608,12 @@ def generate_taxes_w2_and_1099(
         The simulated population has no data for this dataset in the
         specified year or state.
     """
-    user_filters: list[tuple[str, str, Union[str, int]]] = []
+    user_filters: list[tuple[Optional[str], str, Union[str, int]]] = []
     if year is not None:
         user_filters.append((DATASET_SCHEMAS.tax_w2_1099.date_column_name, "==", year))
         seed = seed * 10_000 + year
     if state is not None:
-        state_column_name: str = DATASET_SCHEMAS.tax_w2_1099.state_column_name
+        state_column_name: Optional[str] = DATASET_SCHEMAS.tax_w2_1099.state_column_name
```

Comment:
how can `DATASET_SCHEMAS.tax_w2_1099.state_column_name` be None?

Reply:
Yeah I'm not sure but this is what mypy was saying. @albrja let me know that we know it's not None if state is not None but it doesn't seem like there's a way of letting mypy know that.